Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -108,16 +113,20 @@
               app:layout_constraintTop_toTopOf="parent"
               tools:src="@drawable/avatar_3" />
 
-          <ImageView
-              android:id="@+id/email_star_image_view"
+          <com.google.android.material.button.MaterialButton
+              android:id="@+id/email_star_button"
+              style="@style/Widget.Material3.Button.IconButton"
               android:layout_width="@dimen/email_star_image_size"
               android:layout_height="@dimen/email_star_image_size"
-              android:padding="@dimen/grid_1"
-              android:background="?attr/selectableItemBackgroundBorderless"
-              android:contentDescription="@{email.starred ? @string/email_starred_content_desc : @string/email_unstarred_content_desc}"
-              android:foreground="@drawable/ic_circle"
-              android:src="@{email.starred ? @drawable/ic_star_filled : @drawable/ic_star_border}"
-              app:glideCircularCrop="@{true}"
+              android:checked="@{email.isStarred}"
+              android:checkable="true"
+              android:onClick="@{(view) -> email.toggleStar()}"
+              android:theme="@style/ThemeOverlay.Reply.ElevationOverlaysDisabled"
```

Comment:
hmm how come the elevation overlays are being disabled if we set the `app:backgroundElevationOverlay` to level 1 below?